chamelon/dune.jst

```diff
@@ -10,6 +10,8 @@
 
 (env
   (dev
-    (flags (:standard -warn-error -A))))
+    (flags (:standard -no-principal -warn-error -A -w -70)))
+  (_
+    (flags (:standard -no-principal -w -70))))
```

Comment:
Is this doing anything when we're turning on `-warn-error` in the top-level `dune` anyway?